api/api.go

```diff
@@ -0,0 +1,17 @@
+package api
+
+import (
+	"github.com/gorilla/mux"
+	"github.com/jcaladwizeline/academy-go-q42021/api/controller"
```

Comment:
Imports should be ordered alphabetically and separated into three main blocks (or two, or only one, if other blocks are not presented), which are:

- Go's standard library/packages.
- internal packages.
- Third-party packages.
```suggestion
	"github.com/jcaladwizeline/academy-go-q42021/api/controller"
	
	"github.com/gorilla/mux"
```